docs/rules/no-underscore-dangle.md

```diff
@@ -113,6 +114,20 @@ const o = {
 };
 ```
 
+### allowFunctionParams
+
+Examples of **correct** code for this rule with the `{ "allowFunctionParams": true }` option:
```

Comment:
Since this will be `true` by default, can we add a correct example with a function parameter name with a dangling underscore to the correct section starting on line 27 and make this section show examples of incorrect code for`/*eslint no-underscore-dangle: ["error", { "allowFunctionParams": false }]*/` (in other words, the non-default behavior).